.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - "5"
+  - "4"
+  - "4.2"
+
+cache:
+  directories:
+    - node_modules
+
+before_script:
+  - NODE_ENV='ci'
+
+script:
+  - npm run test:ci
```

Comment:
I wonder if we should just use "npm test" here?

Projects could define "npm run test:unit" or similar for just running the unit tests. It all depends on what we think "npm test" should mean. In the new date-controller repo I've got "npm test" set up to run the linter, the unit tests, and check that the test coverage is high enough.